packages/blocks/cloudhealth/src/lib/auth.ts

```diff
@@ -0,0 +1,23 @@
+import { BlockAuth } from '@openops/blocks-framework';
+
+const markdown = `
```

Comment:
Personally i find it a bit intruding

<img width="540" alt="image" src="https://github.com/user-attachments/assets/651204f0-edec-431f-9ac2-d0addcf977d2" />